cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/LocalizationUtils.java

```diff
@@ -42,6 +48,43 @@
 public final class LocalizationUtils {
   private static final Logger LOG = LoggerFactory.getLogger(LocalizationUtils.class);
 
+  public static List<File> getExtraJars(CConfiguration cConf) {
+    String extraJarConf = cConf.get(Constants.AppFabric.EXTRA_JARS);
+    List<String> extraJars = extraJarConf == null ?
+      new ArrayList<String>() : Arrays.asList(extraJarConf.split(","));
+    List<File> jarFiles = new ArrayList<>();
+    for (String jarPath : extraJars) {
+      jarFiles.add(new File(jarPath));
+    }
+    return jarFiles;
+  }
+
+  public static  List<String> getExtraJarNames(CConfiguration cConf) {
+    List<String> jars = new ArrayList<>();
+    for (File jar : getExtraJars(cConf)) {
+      jars.add(jar.getName());
+    }
+    return jars;
+  }
+
+  /**
+   * Add the extra jars path set in the {@link CConfiguration} in a map.
+   */
+  public static void addExtraJars(Map<String, LocalizeResource> resourcesToLocalize, CConfiguration cConf) {
```

Comment:
Same for this one. If a method is only used in one place, don't make it a static util method. Make it a private method in that class.